README.md

```diff
@@ -46,6 +46,8 @@ All calls to Microsoft Graph are chained together starting with `client.api(path
 * https://graph.microsoft.com/beta/me
 * me/events?$filter=startswith(subject, 'ship')
 
+For more examples of accepted paths, see the [test cases](tests/urlParsing.js).
```

Comment:
I actually just removed this line since we cover all the patterns here and the test file really isn't a great example. Is this a merge issue or did you want to add it back? (The link is wrong, so we'll have to fix that if keeping)